File: client/jetbrains/src/main/java/com/sourcegraph/cody/ui/Colors.java
package com.sourcegraph.cody.ui;

import com.intellij.ui.JBColor;
import com.intellij.util.ui.JBUI;
import java.awt.*;

public class Colors {
  public static final Color CYAN = new JBColor(new Color(0, 203, 236), new Color(0, 203, 236));
  public static final Color PURPLE = new JBColor(new Color(161, 18, 255), new Color(161, 18, 255));
  public static final Color ORANGE = new JBColor(new Color(255, 85, 67), new Color(255, 85, 67));
  public static final Color SECONDARY_LINK_COLOR =
      new JBColor(JBUI.CurrentTheme.Link.Foreground.ENABLED, new Color(168, 173, 189));
}

File: client/jetbrains/src/main/kotlin/com/sourcegraph/cody/ui/Colors.kt
package com.sourcegraph.cody.ui

import com.intellij.ui.JBColor
import com.intellij.util.ui.JBUI
import java.awt.Color

object Colors {

  @JvmField val CYAN: Color = JBColor(Color(0, 203, 236), Color(0, 203, 236))

  @JvmField val PURPLE: Color = JBColor(Color(161, 18, 255), Color(161, 18, 255))

  @JvmField val ORANGE: Color = JBColor(Color(255, 85, 67), Color(255, 85, 67))

  @JvmField
  val SECONDARY_LINK_COLOR: Color =
      JBColor(JBUI.CurrentTheme.Link.Foreground.ENABLED, Color(129, 133, 148))
}
